### ISSUE TITLE
[Game] Caesars Palace Advance - Takes too long to transition to most of the game modes

### ISSUE THREAD

**Issue by Dartz150** [Gericom/GBARunner3#132]
### Game Info
**Game Title:** Caesars Palace Advance - Millennium Gold Edition
**TID:** ACPE
**REG:** USA
**SAVETYPE:** NO_SAVE
**Hicode:** 
**Bitmap:** ✔
### Hardware
**Console:** DSi
**Media:** SD
**Flashcard:** 
### GBARunner3 info
**Commit:** 45161aa
**Build Date:** 11 Dec 2023
**Branch:** develop
**JIT Patches on repo:** no
### Configuration
**Base Shift:** 
**JIT Patches:** ✔
**wram icache:** ✘

 ### **Compatibility:** Unplayable

**Notes:** The game takes too long to transition between any game mode, making it seem to freeze, same results with wram icache off, ewram dcache on and manual JIT patches, it may be timing related.

**Comment by Dartz150** [comment]
As of commit [2f0609f](https://github.com/Gericom/GBARunner3/commit/2f0609f7d3885cf8f5af3a9938f2b53d3e08982d), the game works properly now, but the game mode selection screen plays a screech sound repeatedly, similar to a cash register printer.

**Comment by Gericom** [comment]
Did it have proper sound previously?

**Comment by Dartz150** [comment]
> Did it have proper sound previously?

Yes, I haven't tested this game recently, so is hard to exactly tell when the issue began to occur. I will try to bisect it later.
